INSTALL.md

```diff
@@ -0,0 +1,291 @@
+
+The `User Hostname` field will also only appear if setup is creating the database.  It is the hostname of the Apache/PHP server from which the user, i.e,  `Login Name` is permitted to connect to the MySQL database.  If you are setting up MySQL and Apache/PHP on the same computer, then you can use `localhost`. 
+
+The `UTF-8 Collation` field is the collation setting for mysql. If the language you are planning to use in LibreHealthEHR is in the menu, then you can select it. Otherwise, just select `General`. Choosing`None` is not recommended and will force latin1 encoding.
+
+The `Initial User` is the username of the first user, which is what they will use to login.  Limit this to one word only.
+
+The `Initial User Password` is the password of the user entered into the above `Initial User` field.
+
+The `Initial User's First Name` is the value to be used as their first name.  This information may be changed in the user administration page.
+
+The `Initial User's Last Name` is the value to be used as their last name.  This information may be changed in the user administration page.
+
+The `Initial Group` is the first group, basically name of the practice, that will be created.  A user may belong to multiple groups, which again, can be altered on the user administration page. It is suggested that no more than one group per office be used.
+
+
+#### Step 3
+This is where setup will configure LibreHealthEHR.  It will first create the database and connect to it to create the initial tables.  It will then write the mysql database configuration to the `libreehr/sites/default/sqlconf.php` file. 
+
+Should anything fail during Step 3, you may have to remove the existing database or tables before you can try again. If no errors occur, you will see a `Continue` button at the bottom.
+
+
+#### Step 4
+This step will install and configure the embedded phpGACL access controls.  It will first write configuration settings to files.  It will then configure the database.  It will then give the `Initial User` administrator access. 
+
+Should anything fail during Step 4, you may have to remove the existing database or tables before you can try again. If no errors occur, you will see a `Continue` button at the bottom.
+
+
+#### Step 5
+You will be given instructions on configuring the PHP.  We suggest you print these instructions for future reference.  Instructions are given to edit the `php.ini` configuration file.  If possible, the location of your `php.ini` file will be displayed in green. 
+
+If your `php.ini` file location is not displayed, then you will need to search for it.  The location of the `php.ini` file is dependent on the operating system.  In linux, `php.ini` is generally found in the `/etc/` directory.  In Windows, the `XAMPP 1.7.0` package places the `php.ini` file in the `xampp\apache\bin\` directory. 
+
```

Comment:
Is 1.7.0 the version of XAMPP that is currently being used. May be best to not give a version number